golint.sh

```diff
@@ -1,10 +1,24 @@
 #!/bin/bash
+# Copyright 2020 Red Hat, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
 
-cd "$(dirname $0)"
+cd "$(dirname "$0")" || exit
```

Comment:
Maybe here should be `cd "$(dirname "$0")" || exit 1` ?

Reply:
it is enough to use `|| exit`. Can be checked in nested bash followed by `echo $?`

Reply:
When a script ends with an exit that has no parameter, the exit status of the script is the exit status of the last command executed in the script (previous to the exit).

Reply:
Ok, thank you